conf/ocsci/fdf_pre_ga_deployment.yaml

```diff
@@ -0,0 +1,12 @@
+---
+DEPLOYMENT:
+  fusion_deployment: true
+  fdf_deployment: true
+  channel: v2.0
```

Comment:
If this channel is related to Fusion or FDF I think we should rename it to some more specific value as this is too generic name it like `channel` I prefer `fusion_channel` if it's more related to fusion here.